marlowe-spec-test/README.md

```diff
@@ -1,14 +1,14 @@
 # Marlowe Spec Test
 
-This tool is a test suite that checks if a particular Marlowe implementation conforms to the specification.
+This documentation refers to the Marlowe Test Spec Driver, a test suite that checks that a particular Marlowe implementation conforms to the specification.
 
-The tool receives as an argument the path to a process (with no arguments) that we call "client process". The client process listens for semantic requests over `stdin` and replies via `stdout`, any `stderr` message is just printed.
+The driver receives as an argument the path to a program that we call "client process". The client process is called with no arguments, and it listens for semantic requests over `stdin` and replies via `stdout`, any `stderr` message is treated as debug information.
```

Comment:
the stderr is currently just printed, but I'll make it part of the debug log file eventually.